cmake/project-config.cmake.in

```diff
@@ -11,9 +11,9 @@
 # Tell the user project where to find our headers and libraries
 get_filename_component (_DIR ${CMAKE_CURRENT_LIST_FILE} PATH)
 get_filename_component (_ROOT "${_DIR}/@PROJECT_ROOT_DIR@" ABSOLUTE)
-set (@PROJECT_VARIANT_NAME@_INCLUDE_DIRS "${_ROOT}/@INCLUDEDIR@")
-set (@PROJECT_VARIANT_NAME@_LIBRARY_DIRS "${_ROOT}/@LIBDIR@")
-set (@PROJECT_VARIANT_NAME@_BINARY_DIRS "${_ROOT}/@BINDIR@")
+set (@PROJECT_VARIANT_NAME@_INCLUDE_DIRS ${CMAKE_INSTALL_INCLUDEDIR})
+set (@PROJECT_VARIANT_NAME@_LIBRARY_DIRS ${CMAKE_INSTALL_LIBDIR})
+set (@PROJECT_VARIANT_NAME@_BINARY_DIRS ${CMAKE_INSTALL_BINDIR})
```

Comment:
I believe this should rather be:
```suggestion
set (@PROJECT_VARIANT_NAME@_INCLUDE_DIRS "${_ROOT}/@CMAKE_INSTALL_INCLUDEDIR@")
set (@PROJECT_VARIANT_NAME@_LIBRARY_DIRS "${_ROOT}/@CMAKE_INSTALL_LIBDIR@")
set (@PROJECT_VARIANT_NAME@_BINARY_DIRS "${_ROOT}/@CMAKE_INSTALL_BINDIR@")
```
as _ROOT must remain referenced

Reply:
`CMAKE_INSTALL_<dir>` are relative paths, meant to be used with `install( ... DESTINATION ...)`.

For absolute paths, there is `CMAKE_INSTALL_FULL_<dir>`. This takes into account `CMAKE_INSTALL_PREFIX` (but breaks `CMAKE_STAGING_PREFIX`).

Now what is `_ROOT`?

Reply:
> Now what is _ROOT?

As far as I can see, this is a way to figure out the (actual) installation prefix from where the CMake config file is located.

> If these variables should be the include, library, and binary directories of install, then the `FULL` variables from `GNUInstallDirs` will do fine

yeah that was my first thought too. But it seems the existing logic in the file was to make it possibly relocatable. And I hope that what I proposed would make it still possible

Reply:
Sorry, I missed part of the context. Basically @rouault is right, and I see where `_ROOT` comes from now.

I will test the PR in the next days.